src/core/binding.h

```diff
@@ -225,6 +225,11 @@ typedef struct QUIC_BINDING {
     //
     CXPLAT_LIST_ENTRY Listeners;
 
+    //
+    // The connections registered on this binding.
+    //
+    CXPLAT_LIST_ENTRY Connections;
```

Comment:
This shouldn't be on the binding itself, but as a part of the `Lookup`. Additionally, for performance, we don't want to use a list; that's why the Lookup uses hashtables. You're on the right path, but the lookup hash tables and entries will have to be modified to get this stuff to work.

Reply:
I've implemented multiple hash entries for a Source CID.